### PR TITLE
Deprecate Standalone cinder

### PR DESCRIPTION
**Pull request by adisky**
**What this PR does / why we need it**:
Deprecate standalone cinder

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -36,15 +36,16 @@
...
+    # cloud-provider-openstack-acceptance-test-k8s-cinder:
+    #  jobs:
+    #    - cloud-provider-openstack-acceptance-test-k8s-cinder:
+    #        irrelevant-files:
+    #          - ^docs/.*$
+    #          - ^.*\.md$
+    #          - ^OWNERS$
+    #          - ^SECURITY_CONTACTS$
+    #          - ^.gitignore$
     cloud-provider-openstack-acceptance-test-standalone-cinder:
```
Comment (lingxiankong) [inline .zuul.yaml]: I think we want to disable this one.

Comment (adisky) [reply .zuul.yaml]: Thanks for pointing updated!!

**Pull request merged**